spec.html

```diff
@@ -1508,28 +1509,28 @@ <h1>Number::exponentiate ( _base_, _exponent_ )</h1>
           <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-dependent approximation of the result of raising _base_ to the power _exponent_, subject to the following requirements:</p>
           <ul>
             <li>If _exponent_ is *NaN*, the result is *NaN*.</li>
-            <li>If _exponent_ is *+0*, the result is 1, even if _base_ is *NaN*.</li>
-            <li>If _exponent_ is *-0*, the result is 1, even if _base_ is *NaN*.</li>
+            <li>If _exponent_ is *+0*, the result is *1*<sub>𝔽</sub>, even if _base_ is *NaN*.</li>
```

Comment:
We should qualify ±0 with 𝔽 as well. It's weird to rely on the 0 vs. +0 distinction. What will we do when we introduce Decimal? Will +0 be a Number or Decimal positive zero?

Reply:
I think we need to reach a consensus here. It is clashing with @michaelficarra's previous request to remove subscripts for *+0* and *-0*. I don't have a strong opinion about it and I'm fine with both directions, but it would be good to have a clear path to follow.

Reply:
Editor group talked about it and agrees with @waldemarhorwat; see @michaelficarra's [comment](https://github.com/tc39/ecma262/pull/2007#issuecomment-634985074) below. Apologies for the churn.